pkg/util/validation/limits.go

```diff
@@ -478,6 +481,18 @@ func (o *Overrides) ActiveSeriesCustomTrackersConfig(userID string) activeseries
 	return o.getOverridesForUser(userID).ActiveSeriesCustomTrackersConfig
 }
 
+// OOOAllowance returns the user specified OOO Allowance if it is non-zero.
+// If it is not set or set to 0, then the default OOO Allowance is returned.
+func (o *Overrides) OOOAllowance(userID string) time.Duration {
+	if o.tenantLimits != nil {
+		l := o.tenantLimits.ByUserID(userID)
+		if l != nil && l.OOOAllowance != time.Duration(0) {
+			return l.OOOAllowance
+		}
+	}
+	return o.defaultLimits.OOOAllowance
```

Comment:
```suggestion
    o.getOverridesForUser(userID).OOOAllowance
```

Note that value returned by `o.tenantLimits.ByUserID(userID)` already has either override or default value set.

Reply:
Oh so if default was set to 30m and nothing was specified for the tenant specific override, this will still give us 30m?

Reply:
> Oh so if default was set to 30m and nothing was specified for the tenant specific override, this will still give us 30m?

Yes. The magic happens during YAML unmarshalling, when we set the default value before YAML sets its own values.

Reply:
Another question for the unmarshalling magic: If the default is 30m, and if a tenant is explicitly set to 0s, it will give me 0s, right?

Reply:
> Another question for the unmarshalling magic: If the default is 30m, and if a tenant is explicitly set to 0s, it will give me 0s, right?

That's correct.